### PR TITLE
Add a script to pre-process the data for GMN

### PR DESCRIPTION
**Pull request by boom10899**
I wrote a script for converting the result and dataset from HDF5 format to CSV format that ready to run with GMN.

### CODE REVIEW

**Review by keichi — COMMENTED** [review]
Can you run [flake8](https://flake8.pycqa.org/en/latest/) on this script and reformat? Also please change the file name to snake case (`preprocess_gmn.py`) please.

**Comment by boom10899** [comment]
I reformatted the code follow flake8 and changed the file name. Could you please re-review the script again?

File: scripts/preprocess_gmn.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+
+import numpy as np
```
Comment (keichi) [inline scripts/preprocess_gmn.py:3]: This import seems ununsed.

File: scripts/preprocess_gmn.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+import pandas as pd
+import h5py
+
+import sys
```
Comment (keichi) [inline scripts/preprocess_gmn.py:7]: This one too.

**Review by keichi — COMMENTED** [review]
Can you remove the unused imports? Otherwise it looks good to me.

**Comment by boom10899** [comment]
Thank you very much. I removed unused imports and fix a bug in the code during converting byte to sting.

**Comment by keichi** [comment]
Thanks!

**Pull request merged**